src/test/mochitest/browser_dbg-breaking.js

```diff
@@ -13,7 +13,7 @@ add_task(async function() {
   reload(dbg);
   await waitForPaused(dbg);
 
-  await waitForLoadedSource(dbg, "doc-scripts.html");
+  await waitForSelectedSource(dbg, "doc-scripts.html");
```

Comment:
yikes... perhaps we should audit these `waitForLoadedSource`...

Reply:
yikes yes you are right